src/components/features/Brand/Brand.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Brand.module.scss';
+
+class Brand extends React.Component {
+  render() {
+    const { brands } = this.props;
+
+    return (
+      <div className={styles.root}>
+        <div className={styles.component}>
+          <div className='container'>
+            <div className={`row ${styles.marksRow}`}>
+              {brands.map(item => (
+                <div key={item.id} className={`col ${styles.mark}`}>
+                  <img className={styles.photo} alt='brand_mark' src={item.mark}></img>
+                </div>
```

Comment:
Na layoucie obecne są również strzałki służące do przesuwania w lewo oraz prawo, a także górny oraz dolny border wyznaczający granicę sekcji

Reply:
to z funkcjonalmności tego komponentu, osobny task..

Reply:
buttony oraz bordery należą moim zdaniem do elementów layoutu, akcje wykonywane przy kliknięciu będą dotyczyć funkcjonalności ;)

Reply:
ok, dodałem :)
wykorzystałem elementy z galerii